src/openrct2-ui/scripting/CustomWindow.cpp

```diff
@@ -228,6 +221,12 @@ namespace OpenRCT2::Ui::Windows
                     result.offset = { AsOrDefault(dukCoord["x"], 0), AsOrDefault(dukCoord["y"], 0) };
                 }
             }
+            else if (dukImage.type() == DukValue::Type::STRING || dukImage.type() == DukValue::Type::NUMBER)
```

Comment:
You swapped the if and else around, which doesn’t change how it works but causes an unnecessarily large diff. Please change the order back.